tests/Export/DynamicProductGroupServiceTest.php

```diff
@@ -304,4 +298,33 @@ private function createProducts(): array
 
         return $products;
     }
+
+    private function getProductStreamCategories(Context $context): array
```

Comment:
Add a docblock of the return type, so the IDE already knows what data is returned here.